pom.xml

```diff
@@ -20,6 +20,33 @@
             <version>4.13.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
you can simplify your dependencies, spring-webmvc already contains spring-core, beans, context